trackintel/io/postgis.py

```diff
@@ -11,6 +11,7 @@
 
 import trackintel as ti
 from trackintel.io.util import _index_warning_default_none
+from trackintel.model.util import doc, _shared_docs
```

Comment:
This will not cause a circular import problem?

Reply:
Hopefully not. :D It feels like untying a complicated knot, pulling at one place creates problems elsewhere.
The idea is that `trackintel.model.util` is not dependent on any `trackintel` module. 
This allows it to be imported anywhere without creating any circular dependencies.
Further `trackintel.model.positionfixes` ... `tours` etc. will only import `trackintel.model.util` thus being themselves circle free in any context. At least that is the idea.